lib/RunTests.js

```diff
@@ -284,6 +289,10 @@ function runTests(
         });
         if (currentRun === undefined) {
           progressLogger.clearConsole();
+          if (preserveOutput) {
+            console.log(chalk.blue('\n\n\nRerunning tests due to changes...'));
+            console.log(chalk.blue('=================================\n\n\n'));
```

Comment:
The intention is to have the `====` perfectly underline the message, right? If so, something like this might make that more clear:

```suggestion
            const message = 'Rerunning tests due to changes...';
            const separator = '='.repeat(message.length);
            console.log(chalk.blue(`\n\n\n${message}\n${separator}\n\n\n`));
```

Reply:
yes, makes sense